assets/helpers/tracking/googleTagManager.js

```diff
@@ -23,6 +26,58 @@ function getDataValue(name, generator) {
   return value;
 }
 
+function getPaymentAPIStatus(): Promise<PaymentRequestAPIStatus> {
+  return new Promise((resolve) => {
+    try {
+
+      const { PaymentRequest } = window;
+
+      if (typeof PaymentRequest !== 'function') {
+        resolve('PaymentRequestAPINotAvailable');
+      }
+
+      const supportedInstruments = [
+        {
+          supportedMethods: 'basic-card',
+          data: {
+            supportedNetworks: ['visa', 'mastercard', 'amex', 'jcb',
+              'diners', 'discover', 'mir', 'unionpay'],
+            supportedTypes: ['credit', 'debit'],
+          },
+        },
+      ];
+
+      const details = {
+        total: {
+          label: 'tracking',
+          amount:
+            {
+              value: '1',
+              currency: 'GBP',
```

Comment:
Does specifying GBP have any impact on the ability to make a payment if the user is in the US for example?

Reply:
Good questions, I don't think so (I tested the feature by putting a dummy credit card and it detected correctly). Nevertheless, I switched to use the `getCurrency` method in badfba3